tests/test_search.py

```diff
@@ -132,9 +134,10 @@ def test_search_returns_N_results_when_N_documents_match_query(self):
         """Create N identical documents. A search query is executed to match the document. All documents created must be
         present in the query results. N is varied across a variety of values.
         """
-        test_matches = [0, 1, 9, 10, 11, 1000, 5000]
+        test_matches = [0, 1, 9, 10, 11, 1000, 10001]
```

Comment:
why this change?

Reply:
normal searching only returns up to the first 10000 results. This check that we can exceed that limit using paging.